### ISSUE TITLE
Create option to hide the blue main menu bar on top

### ISSUE THREAD

**Issue by cwtuan** [josdejong/jsoneditor#581]
```
var editor = new JSONEditor(dom, {
                            mode: 'code',
                            navigationBar: false,
                            statusBar: false,
                        });

```
statusBar is hidden, but navigationBar isn't.

![image](https://user-images.githubusercontent.com/3455798/46075606-96560680-c1bd-11e8-9023-f4f720696b52.png)

**Comment by josdejong** [comment]
The navigation bar this the bar beneath the main menu:

![image](https://user-images.githubusercontent.com/568626/46104143-e9868400-c1d2-11e8-9e2c-04172080b5c3.png)

If you want to hide the main menu I think you have to be a bit creative with CSS

**Comment by cwtuan** [comment]
It would be nice if `mainMenuBar` option is supported.

**Comment by josdejong** [comment]
Yes ok we can do that. I will change the title accordingly

**Comment by cwtuan** [comment]
Thanks

**Comment by josdejong** [comment]
Forgot to close this issue, feature was implemented via #596
